src/core/dev/hw_queue_rx.cpp

```diff
@@ -198,26 +198,32 @@ bool hw_queue_rx::prepare_doca_rxq()
         return false;
     }
 
+    err = doca_pe_set_event_mode(pe, DOCA_PE_EVENT_MODE_PROGRESS_ALL);
+    if (DOCA_IS_ERROR(err)) {
+        PRINT_DOCA_ERR(hwqrx_logerr, err, "doca_pe_set_event_mode pe: %p", pe);
+        return false;
+    }
+
 #if defined(DEFINED_NGINX) || defined(DEFINED_ENVOY)
     // For some scenario with forking usage we may want to distribute CQs across multiple
     // CPUs to improve CPS in case of multiple processes.
 
     // TODO: [DOCA] Replace with real information from DOCA.
-    int comp_channel_num = 1U;
+    //int comp_channel_num = 1U;
 
     // TODO: [DOCA] Currently not functional.
-    if (safe_mce_sys().app.distribute_cq_interrupts && g_p_app->get_worker_id() >= 0 &&
-        m_p_ib_ctx_handler->is_notification_affinity_supported()) {
-        uint32_t comp_ch = g_p_app->get_worker_id() % comp_channel_num;
-        hwqrx_logdbg("Setting PE completion affinity: %" PRIu32 ", pid: %d", comp_ch, getpid());
-
-        err = doca_pe_set_notification_affinity(pe, comp_ch);
-        if (DOCA_IS_ERROR(err)) {
-            PRINT_DOCA_ERR(hwqrx_logerr, err,
-                           "doca_pe_set_notification_affinity pe/ctx/rxq: %p,%p,%p", pe,
-                           m_doca_ctx_rxq, m_doca_rxq.get());
-        }
-    }
+    //if (safe_mce_sys().app.distribute_cq_interrupts && g_p_app->get_worker_id() >= 0 &&
```

Comment:
Why not removing these lines?

Reply:
Just not to forget. There is a PR from Tomer that changes it anyway.

Reply:
On a second thought i will remove this code